tests/integration/components/bs-modal/header-test.js

```diff
@@ -18,37 +18,37 @@ module('Integration | Component | bs-modal/header', function(hooks) {
   });
 
   test('Header can be rendered as non-block', async function(assert) {
-    await render(hbs`{{bs-modal/header title="Test"}}`);
+    await render(hbs`<BsModal::Header @title="Test" />`);
 
     assert.dom('.modal-header').exists({ count: 1 }, 'Modal header is rendered.');
     assert.equal(this.element.querySelector('.modal-header .modal-title').innerHTML.trim(), 'Test', 'Title is shown.');
     assert.dom('.modal-header button.close').exists({ count: 1 }, 'Modal header has close button.');
   });
 
   test('Header can have custom block content', async function(assert) {
-    await render(hbs`{{#bs-modal/header}}<div id="custom">Test</div>{{/bs-modal/header}}`);
+    await render(hbs`<BsModal::Header><div id="custom">Test</div></BsModal::Header>`);
 
     assert.dom('.modal-header div#custom').exists({ count: 1 }, 'Modal header custom block.');
     assert.equal(this.element.querySelector('.modal-header #custom').innerHTML.trim(), 'Test', 'Block content is shown.');
     assert.dom('.modal-header button.close').exists({ count: 1 }, 'Modal header has close button.');
   });
 
   test('Header can yield components when block', async function(assert) {
-    await render(hbs`{{#bs-modal/header as |header|}}{{#header.title}}<div id="custom">Test</div>{{/header.title}}{{header.close}}{{/bs-modal/header}}`);
+    await render(hbs`<BsModal::Header as |header|><header.title><div id="custom">Test</div></header.title>{{header.close}}</BsModal::Header>`);
 
     assert.dom('.modal-header div#custom').exists({ count: 1 }, 'Modal header custom block.');
     assert.equal(this.element.querySelector('.modal-header #custom').innerHTML.trim(), 'Test', 'Block content is shown.');
     assert.dom('.modal-header button.close').exists({ count: 1 }, 'Modal header has close button.');
   });
 
   test('close button can be removed', async function(assert) {
-    await render(hbs`{{bs-modal/header title="Header" closeButton=false}}`);
+    await render(hbs`<BsModal::Header @title="Header" @closeButton={{false}} />`);
 
     assert.dom('.modal-header button.close').doesNotExist('Modal header has no close button.');
   });
 
   test('close button can be removed in yield block form', async function(assert) {
-    await render(hbs`{{#bs-modal/header as |header|}}<div id="custom">Test</div>{{/bs-modal/header}}`);
+    await render(hbs`<BsModal::Header><div id="custom">Test</div></BsModal::Header>`);
```

Comment:
This test is failing now:

```
not ok 653 Chrome 79.0 - [54 ms] - Integration | Component | bs-modal/header: close button can be removed in yield block form

    ---

        actual: >

            Element .modal-header button.close exists once

        expected: >

            Element .modal-header button.close does not exist

        stack: >

                at DOMAssertions.exists (http://localhost:7357/assets/test-support.js:40540:16)

                at DOMAssertions.doesNotExist (http://localhost:7357/assets/test-support.js:40848:18)

                at Object._callee6$ (http://localhost:7357/assets/tests.js:18278:58)

                at tryCatch (http://localhost:7357/assets/vendor.js:413:40)

                at Generator.invoke [as _invoke] (http://localhost:7357/assets/vendor.js:687:22)

                at Generator.prototype.<computed> [as next] (http://localhost:7357/assets/vendor.js:446:21)

        message: >

            Modal header has no close button.

        negative: >

            false

        browser log: |

    ...
```

I don't see how that should be related to this change. :confused:

Reply:
`Integration | Component | bs-form/element` is also failing on a strange way. Both `supported input attributes propagate` and `supported textarea attributes propagate` are failing for `readonly` attribute only. :dizzy_face:

Reply:
Not sure what's happening for `readonly`. But as this is going to be removed anyway, as part of the deprecated attributes, maybe we should just delete these tests here, and stop caring!? :)

Reply:
Yeah, lets be pragmatic. Enough work still left to do.